api/dbadapters/sqldatawarehouse.ts

```diff
@@ -11,6 +11,10 @@ const TABLE_TYPE_COL_NAME = "table_type";
 const COLUMN_NAME_COL_NAME = "column_name";
 const DATA_TYPE_COL_NAME = "data_type";
 const IS_NULLABLE_COL_NAME = "is_nullable";
+const DB_CONNECTION_TIMEOUT = 300000; //5 minute connection timeout
```

Comment:
```suggestion
const DB_CONNECTION_TIMEOUT_MILLIS = 5 * 60 * 1000; //5 minute connection timeout
```

(similar suggestion below)